tools/clang/lib/SPIRV/RemoveBufferBlockVisitor.cpp

```diff
@@ -15,7 +15,7 @@ namespace clang {
 namespace spirv {
 
 bool RemoveBufferBlockVisitor::isBufferBlockDecorationDeprecated() {
```

Comment:
optional style nit: This should preferably be rewritten as `isBufferBlockDecorationAvailable()` then negate its usages (so as not to equate "deprecated" and "removed", and to avoid double negatives).

Reply:
Would you then negate it here? We can't check for `isTargetEnvSpirv1.3OrBelow` (if that function is added) since `SPV_ENV_VULKAN_1_1` (etc) is a higher enum-value than `SPIRV_1_3_UNIVERSAL`, but still counts valid for `1_3`.

In other words, what would you turn this function body into? `return !featureManager.isTargetEnvSpirv1p4OrAbove();`?